openwisp_controller/config/tests/test_vpn.py

```diff
@@ -419,3 +419,15 @@ def test_update_vpn_dh(self, dhparam):
         vpn.refresh_from_db()
         self.assertNotEqual(vpn.dh, Vpn._placeholder_dh)
         dhparam.assert_called_once()
+
+    def test_multiple_vpn_clients(self):
```

Comment:
The code is in the config model so it's better to move it to `test_config`. Keep in mind to avoid adding tests to classes which inherit from `TransactionTestCase` unless it's really needed.

Reply:
Noted  👍🏼